Lecture-6/Task1/l6task1/factories/CoffeeFactory.java

```diff
@@ -0,0 +1,25 @@
+package l6task1.factories;
+
+import l6task1.Coffee;
+import l6task1.CoffeeSort;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CoffeeFactory {
+
+    public static List<Coffee> createCoffees(CoffeeSort coffeeSort) {
+        List<Coffee> coffees = new ArrayList<>();
```

Comment:
Redundant variable - use `return new ArabicaCoffeeFactory().createCoffee();` in case statements, 
return `Collections.emptyList()` in the end of method

Reply:
done